clarity_ext/domain/analyte.py

```diff
@@ -1,54 +1,54 @@
 from clarity_ext.domain.container import PlatePosition, Well, Container
+from clarity_ext.utils import get_and_apply
 
 
 class Analyte(object):
     """
-    Describes an Analyte in the Clarity LIMS system, including custom UDFs.
+    Describes an Analyte in the Clarity LIMS system.
 
-    Takes an analyte resource as input.
+    Expects certain mappings to UDFs in clarity. These are provided
+    in udf_map, so they can be overridden in different installations.
     """
 
-    def __init__(self, resource, container):
```

Comment:
Is kwargs restricted to udfs? In that case it should be indicated by the paramter name.